js/modules/accessibility/options/options.js

```diff
@@ -215,6 +218,31 @@ var options = {
          * @apioption accessibility.description
          */
 
+        /**
+         * The ID of an HTML element describing the contents of the chart.
```

Comment:
Using id's doesn't scale well. If we have 10 charts on the page, it would  be preferable to avoid using id's, but rather rely on the HTML structure. For example, default to the next sibling of the container that has a `.highcharts-caption` tag. Maybe we should consider that the default `linkedDescription` be a query selector, but in that case it needs to be relative to the `container`'s parent node, since we can't currently select the parent directly.

Reply:
The caption texts defined in JS are using the `caption` feature, while the ones in HTML are changed to use the `linkedDescription`. I am thinking for the rest of the demos we might want to only use the `linkedDescription` since we consider it better practice. Thoughts on that?

As for IDs, I am not sure if it is safe to make it relative to the parent node of the container. I can imagine users using a CMS where the description is somewhere else in the DOM. Or maybe we can make it relative to the parent node, and also accept an HTML element as parameter - allowing users to make their own fancy queries. E.g:

```javascript
accessibility: {
    linkedDescription: '.highcharts-caption',
    linkedDescription: document.getElementById('myIDForElementSomewhereFarAway')
}
```

Thoughts?